apps/src/lab2/views/dialogs/ShareDialog.tsx

```diff
@@ -78,6 +87,41 @@ const AfeCareerTourBlock: React.FunctionComponent = () => {
   );
 };
 
+const SubmitButtonInfo: React.FunctionComponent<{
+  submissionStatus: string | undefined;
+  dispatch: AppDispatch;
+}> = ({submissionStatus, dispatch}) => {
+  if (!experiments.isEnabled(experiments.LAB2_SUBMIT_PROJECT)) {
```

Comment:
Can we use `isEnabledAllowingQueryString` so that we can do this with a one-off query parameter as well?